hub.go

```diff
@@ -365,6 +365,34 @@ func (hub *Hub) Flush(timeout time.Duration) bool {
 	return client.Flush(timeout)
 }
 
+// Continue a trace based on HTTP header values. If performance is enabled this
+// returns a SpanOption that can be used to start a transaction, otherwise nil.
+// Uses the Global Hub.
+func ContinueTrace(trace, baggage string) (SpanOption, error) {
```

Comment:
We've seen a lot of issues in the past with methods operating on the global hub, hence I would not add this.